frontend/packages/helm-plugin/src/components/forms/install-upgrade/HelmInstallUpgradePage.tsx

```diff
@@ -232,7 +237,7 @@ const HelmInstallUpgradePage: React.FunctionComponent = () => {
       onNamespaceChange={handleNamespaceChange}
       hideApplications
     >
-      <DocumentTitle>{config.title}</DocumentTitle>
+      <DocumentTitle>{config?.title || ''}</DocumentTitle>
```

Comment:
i think we wouldn't want to set the title as empty string

```suggestion
      {config?.title && <DocumentTitle>{config.title}</DocumentTitle> }
```